lib/index.js

```diff
@@ -1,7 +1,6 @@
-'use strict'
+import {Transform} from 'node:stream'
+import {Buffer} from 'node:buffer'
 
-const stream = require('stream')
-const {Buffer} = require('buffer')
 const td = new TextDecoder('utf8', {fatal: true, ignoreBOM: true})
```

Comment:
Note: I think this is still needed for NativeScript, which I don't think highly of.

Reply:
Don't understand... sry

Reply:
That was more a note to myself, sorry.  People using NativeScript probably still have to use `@cto.af/textdecoder` or similar to make this part work.

Reply:
Ah, i see